src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -149,12 +149,23 @@ TEST_F(ResetGlobalPropertiesRequestTest, Run_InvalidVrHelp_UNSUCCESS) {
       mobile_apis::GlobalProperty::KEYBOARDPROPERTIES;
 
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
-  smart_objects::SmartObject so_prompt =
+
+  std::vector<std::string> help_prompt{"help_text"};
```

Comment:
@dev-gh oficially not. I've removed brace initialization in https://github.com/smartdevicelink/sdl_core/pull/1555/commits/a5b4a71170dd4266d19774aecc787cbc91f8bb08